extensions/interactions/NumericInput/protractor.js

```diff
@@ -32,6 +32,8 @@ var expectInteractionDetailsToMatch = function(elem) {
 var submitAnswer = function(elem, answer) {
   elem.element(by.tagName('oppia-interactive-numeric-input')).
     element(by.tagName('input')).sendKeys(answer + '\n');
```

Comment:
So, looking at the old code, the md-button in text/numeric input does not have a "type="submit"" or anything bound to it. I wonder if that's why it isn't calling.

Here is something t hat may be worth trying. Reinstate the form, but try adding a type="submit" to the button. See whether the button now works.

Also, did submitting by pressing Enter ever work?

I'm also fine with a solution that doesn't use a form, but you need some way to preserve the existing "enter key" behaviour.